config.yml.example

```diff
@@ -6,7 +6,7 @@ user: git
 # You only have to change the default if you have configured Unicorn
```

Comment:
Actually, now that there isn't a `config.yml`, the install command on line 25 of the dockerfile fails. Looking into it.